### PR TITLE
shoppingcart

### PR DESCRIPTION
**Pull request by HiteshVagQa**
Please review shopping cart link test

### CODE REVIEW

File: src/test/java/resources/featurefile/shoppingCartPage.feature [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Feature: Shopping cart link test
+  As a user I should navigate to shopping cart and verify the page
+
+  @smoke
+  Scenario: User should be able to navigate to shopping cart link
+    Given I am on HomePage page
+    When I click on shopping cart link
```
Comment (ghost) [inline src/test/java/resources/featurefile/shoppingCartPage.feature]: Your code is not on same line .Please keep same distance .

**Review by HiteshVagQa — COMMENTED** [review]
All Okay

**Review by HiteshVagQa — COMMENTED** [review]
All Okay

File: src/test/java/resources/featurefile/shoppingCartPage.feature (hunk 1 above)
Comment (HiteshVagQa) [reply src/test/java/resources/featurefile/shoppingCartPage.feature]: resolved

File: src/main/java/com/nopcommerce/demo/pages/ShoppingCartPage.java [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    WebElement shopppingCartText;
+
+    public void clickOnShoppingCartLink(){
+        driver.findElement((By) shoppingCartLink).click();
+    }
+
+    public void verifyShoppingCartText(){
+        String expectedText = "Shopping cart";
+        String actualText = driver.findElement((By) shopppingCartText).getText();
+        Assert.assertEquals(actualText,expectedText,"Shopping cart page not found");
```
Comment (SaiLaxmiRaj) [inline src/main/java/com/nopcommerce/demo/pages/ShoppingCartPage.java:34]: Please, Can you add log

**Review by bhavika48 — APPROVED** [review]

**Pull request merged**